frontend/src/pages/DashboardPage/TourGuide/TourGuide.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import Joyride from 'react-joyride';
+
+const TourGuide = ({ runTour }) => {
```

Comment:
Is there a way to make this more dynamic instead of hard-coded. Let's say we want a tour for other pages. Will we need a new file or is it usable?